src/sentry/auth/superuser.py

```diff
@@ -275,12 +298,38 @@ def set_logged_in(self, user, current_datetime=None):
         request = self.request
         if current_datetime is None:
             current_datetime = timezone.now()
+
+        token = get_random_string(12)
+
+        if request.method == "PUT":
+            try:
+                # Can't do this through request.data as in auth_index,  request obj is switched to httprequest
+                su_access_json = json.loads(request.body)
+                su_access_info = SuperuserAccessSerializer(data=su_access_json)
+
+                if not su_access_info.is_valid():
+                    raise serializers.ValidationError(su_access_info.errors)
+
+                logger.info(
+                    "superuser.superuser_access",
+                    extra={
+                        "superuser_session_id": token,
+                        "user_id": request.user.id,
+                        "user_email": request.user.email,
+                        "su_access_category": su_access_info.validated_data["categoryOfSUAccess"],
+                        "reason_for_su": su_access_info.validated_data["reasonForSU"],
+                    },
+                )
+            except json.JSONDecodeError as err:
```

Comment:
what's the purpose of this `except` block?

Reply:
I put that incase `json.loads(request.body)` errored out, i haven't seen it happen so i'm not sure if it was needed